components/feature/prompts/src/main/res/layout/mozac_feature_prompt_login_prompt.xml

```diff
@@ -15,19 +15,27 @@
     android:paddingBottom="16dp"
     tools:ignore="Overdraw">
 
+    <ImageView
+        android:id="@+id/host_icon"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/mozac_ic_globe"
+        android:importantForAccessibility="no" />
```

Comment:
If we want the icon to have `textColorPrimary` by default we can use  `app:tint="?android:attr/textColorPrimary"` :)


We are doing something similar on the add-on layout.

  ```xml
  <androidx.appcompat.widget.AppCompatImageView
        android:id="@+id/add_button"
        android:layout_width="48dp"
        android:layout_height="48dp"
        android:scaleType="center"
        android:layout_marginStart="0dp"
        android:layout_marginEnd="6dp"
        android:layout_alignParentEnd="true"
        android:layout_centerVertical="true"
        android:background="?attr/selectableItemBackgroundBorderless"
        android:contentDescription="@string/mozac_feature_addons_install_addon_content_description"
        app:srcCompat="@drawable/mozac_ic_new"
        app:tint="?android:attr/textColorPrimary" />
```

Reply:
we're doing this here as well but possibly it doesn't work? https://github.com/mozilla-mobile/android-components/pull/7705/files#diff-003c19ba3f63e37c94b8020a6ee8b450L29

https://github.com/mozilla-mobile/fenix/issues/8577

Reply:
Based on Emily's comment above and the fact that setting a tint in xml for the ImageView will apply it to the website favicon also we went with applying the tint programatically, just for the default image.
Also looked into setting `fillColor` for the `mozac_ic_globe` (default image) to be `?android:attr/textColorPrimary` since we are the only ones using it but that svg seems to be part of a series of images, all using `mozac_ui_icons_fill` so, again, back to setting a tint programatically.